dockerfiles/theia-dev/docker/alpine/setup-yarn.dockerfile

```diff
@@ -1 +1,10 @@
-# nothing to do on alpine
+ENV YARN_VERSION=1.21.1
```

Comment:
maybe we just need to bump to https://github.com/nodejs/docker-node/blob/eb89a38195d4e289812204847baf06c6c53406c0/10/alpine3.9/Dockerfile

10.19.0-alpine3.11

Reply:
It works. Thanks!